Makefile

```diff
@@ -14,8 +14,9 @@ install:
 
 concourse_e2e:
 	@echo "Executing e2e automated tests against the staging environment..."
-	# execute all scenarios except no submission ( the '-' character prefix means skip)
-	behave behave/features --stop --tags=-e2e_happy_path_no_nhs_login_no_submission
+	# execute all scenarios except no submission and backend submissions test ( the '-' character prefix means skip)
+	behave behave/features --stop --tags="-@e2e_happy_path_no_nhs_login_no_submission and -@simple_web_submission_entry \
```

Comment:
this command is not excluding the `@s3_outputs_check test`
The `behave` tag expressions documentation might be of some use:
https://behave.readthedocs.io/en/latest/behave.html#tag-expression
https://behave.readthedocs.io/en/latest/tag_expressions.html